### DISCUSSION TITLE
How can I enable the userChrome.tabbar.on_bottom.menubar_on_top?

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [black7375/Firefox-UI-Fix#612] [Q&A] [answered]
How can I enable the `userChrome.tabbar.on_bottom.menubar_on_top`? Should I have enabled the `userChrome.tabbar.on_bottom` too? If I should have enabled `userChrome.tabbar.on_bottom` too, then the tab bar overlaps the menu bar, and that is a thing for a new issue.

**Comment by black7375** [comment, marked as answer]
You can only activate `userChrome.tabbar.on_bottom.menubar_on_top` when using `userChrome.tabbar.on_bottom`.

In other cases, you can activate menubar after right-clicking the toolbar.

**Reply by Aleksandra0056** [reply]
Then, the tabs bar overlaps a menu bar, and that is a serious issue. I am going to file a new bug.
